wafer/sponsors/serializers.py

```diff
@@ -0,0 +1,47 @@
+from rest_framework import serializers
+from reversion import revisions
+
+from wafer.sponsors.models import Sponsor, SponsorshipPackage
+
+
+class SponsorSerializer(serializers.ModelSerializer):
+
+    class Meta:
+        model = Sponsor
+        exclude = ('_description_rendered', )
+
+    @revisions.create_revision()
+    def create(self, validated_data):
+        revisions.set_comment("Created via REST api")
+        return super(SponsorSerializer, self).create(validated_data)
+
+    @revisions.create_revision()
+    def update(self, sponsor, validated_data):
+        revisions.set_comment("Changed via REST api")
+        sponsor.name = validated_data['name']
+        sponsor.description = validated_data['description']
```

Comment:
Do we want to allow `number_available`, `currency`, `price` and `short_description` to be updated too?